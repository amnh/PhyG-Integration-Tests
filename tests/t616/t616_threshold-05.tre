(AlbanianTosk:53.0,(Icelandic:46.0,((AncientIonicGreek:28.0,((Danish:23.0,OldPrussian:27.0)HTU18:2.0,(Cornish:27.0,Manx:16.0)HTU21:11.0)HTU23:6.0)HTU17:10.0,((Faroese:20.0,((EasternFrisianSaterland:26.0,Scots:30.0)HTU24:4.0,(Irish:34.0,(KolschColognian:38.0,Yiddish:21.0)HTU25:16.0)HTU26:6.0)HTU22:7.0)HTU19:5.0,(ArmenianWestern:18.0,(Assamese:20.0,SanskritVedic:15.0)HTU29:18.0)HTU28:8.0)HTU27:7.0)HTU20:10.0)HTU30:40.0)HTU16[698.0];
(AlbanianTosk:50.0,(Scots:25.0,(EasternFrisianSaterland:20.0,(Faroese:16.0,(((ArmenianWestern:20.0,(Assamese:18.0,SanskritVedic:12.0)HTU24:16.0)HTU20:13.0,(AncientIonicGreek:48.0,(Irish:26.0,((Danish:24.0,OldPrussian:26.0)HTU26:7.0,(Cornish:25.0,Manx:14.0)HTU27:10.0)HTU29:8.0)HTU22:3.0)HTU28:9.0)HTU17:3.0,(Yiddish:22.0,(Icelandic:42.0,KolschColognian:31.0)HTU23:13.0)HTU18:9.0)HTU19:7.0)HTU21:9.0)HTU25:9.0)HTU30:31.0)HTU16[698.0];
(AlbanianTosk:3.0,(Icelandic:47.0,((((Danish:26.0,OldPrussian:27.0)HTU21:6.0,(Cornish:25.0,Manx:15.0)HTU23:11.0)HTU18:9.0,(((KolschColognian:30.0,Yiddish:18.0)HTU28:7.0,(ArmenianWestern:20.0,(Assamese:19.0,SanskritVedic:13.0)HTU27:16.0)HTU29:9.0)HTU19:6.0,(Faroese:18.0,(AncientIonicGreek:38.0,Irish:25.0)HTU25:12.0)HTU24:2.0)HTU22:5.0)HTU17:5.0,(EasternFrisianSaterland:21.0,Scots:26.0)HTU26:7.0)HTU20:11.0)HTU30:4.0)HTU16[697.0];
(AlbanianTosk:41.0,(Icelandic:48.0,((Scots:26.0,(EasternFrisianSaterland:37.0,((Faroese:17.0,((KolschColognian:38.0,Yiddish:18.0)HTU28:15.0,(AncientIonicGreek:38.0,Irish:22.0)HTU29:9.0)HTU26:5.0)HTU20:7.0,((Danish:26.0,OldPrussian:23.0)HTU22:7.0,(Cornish:25.0,Manx:16.0)HTU27:10.0)HTU25:8.0)HTU17:2.0)HTU19:6.0)HTU18:8.0,(ArmenianWestern:19.0,(Assamese:19.0,SanskritVedic:14.0)HTU23:21.0)HTU24:10.0)HTU21:10.0)HTU30:27.0)HTU16[697.0];
(AlbanianTosk:50.0,(Icelandic:47.0,(Scots:23.0,(EasternFrisianSaterland:21.0,(((ArmenianWestern:21.0,(Assamese:17.0,SanskritVedic:13.0)HTU25:26.0)HTU21:8.0,(KolschColognian:32.0,Yiddish:18.0)HTU22:9.0)HTU19:8.0,(Faroese:21.0,(((Danish:26.0,OldPrussian:23.0)HTU18:8.0,(Cornish:23.0,Manx:18.0)HTU24:10.0)HTU23:6.0,(AncientIonicGreek:38.0,Irish:28.0)HTU27:7.0)HTU20:5.0)HTU29:2.0)HTU26:5.0)HTU28:13.0)HTU17:19.0)HTU30:38.0)HTU16[695.0];
(AlbanianTosk:50.0,(Icelandic:48.0,(Scots:21.0,(EasternFrisianSaterland:20.0,(Faroese:13.0,((ArmenianWestern:18.0,(Assamese:18.0,SanskritVedic:14.0)HTU28:17.0)HTU24:7.0,((KolschColognian:40.0,Yiddish:20.0)HTU17:12.0,(Irish:19.0,(AncientIonicGreek:27.0,((Cornish:28.0,Manx:17.0)HTU25:8.0,(Danish:24.0,OldPrussian:22.0)HTU27:4.0)HTU22:22.0)HTU21:6.0)HTU23:9.0)HTU29:4.0)HTU19:7.0)HTU18:4.0)HTU20:10.0)HTU26:17.0)HTU30:38.0)HTU16[695.0];
(AlbanianTosk:2.0,(Scots:19.0,(Faroese:18.0,(EasternFrisianSaterland:27.0,(((Yiddish:22.0,(Icelandic:48.0,KolschColognian:30.0)HTU24:11.0)HTU22:9.0,(ArmenianWestern:19.0,(Assamese:25.0,SanskritVedic:13.0)HTU18:16.0)HTU23:10.0)HTU17:5.0,(((Cornish:23.0,Manx:15.0)HTU27:10.0,(Danish:26.0,OldPrussian:23.0)HTU28:7.0)HTU20:6.0,(AncientIonicGreek:38.0,Irish:25.0)HTU25:5.0)HTU21:6.0)HTU19:4.0)HTU29:4.0)HTU26:9.0)HTU30:4.0)HTU16[695.0];
