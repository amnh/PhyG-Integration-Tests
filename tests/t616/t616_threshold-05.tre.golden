(AlbanianTosk:71.0,(Icelandic:53.5,((ArmenianWestern:27.5,(Assamese:23.5,SanskritVedic:21.5)HTU23:28.5)HTU22:23.0,(Scots:26.0,(Faroese:23.5,(EasternFrisianSaterland:30.0,(((Cornish:26.5,Manx:23.5)HTU26:18.5,(AncientIonicGreek:41.0,(Danish:31.0,OldPrussian:27.0)HTU19:14.0)HTU29:18.0)HTU17:16.5,(Irish:40.0,(KolschColognian:41.5,Yiddish:30.5)HTU28:20.5)HTU27:13.0)HTU20:10.5)HTU25:13.0)HTU21:19.0)HTU24:23.0)HTU18:27.0)HTU30:65.0)HTU16[696.0];
(AlbanianTosk:27.0,(Icelandic:35.5,(KolschColognian:38.5,(Yiddish:27.5,((Faroese:19.5,((EasternFrisianSaterland:29.0,Scots:34.0)HTU18:15.0,(Irish:29.0,((AncientIonicGreek:37.5,(Danish:31.5,OldPrussian:26.5)HTU23:19.5)HTU22:19.5,(Cornish:25.0,Manx:25.0)HTU26:19.0)HTU19:18.5)HTU25:17.5)HTU27:15.5)HTU17:15.5,(ArmenianWestern:26.5,(Assamese:23.5,SanskritVedic:21.5)HTU29:28.0)HTU28:21.0)HTU20:21.0)HTU21:22.0)HTU24:21.5)HTU30:27.0)HTU16[696.0];
(AlbanianTosk:74.5,(Icelandic:52.5,(Scots:29.0,((ArmenianWestern:28.0,(Assamese:22.5,SanskritVedic:22.5)HTU22:27.0)HTU17:19.5,((Faroese:22.5,(EasternFrisianSaterland:26.5,(Irish:33.5,(KolschColognian:45.0,Yiddish:27.0)HTU24:23.5)HTU28:18.0)HTU25:15.0)HTU23:10.0,(AncientIonicGreek:55.0,((Cornish:30.0,Manx:20.0)HTU26:16.5,(Danish:27.5,OldPrussian:30.5)HTU29:11.5)HTU19:11.5)HTU27:13.0)HTU18:13.5)HTU20:17.5)HTU21:25.5)HTU30:62.0)HTU16[694.0];
